i3pystatus/net_speed.py

```diff
@@ -12,23 +12,56 @@
 class NetSpeed(IntervalModule):
     """
     Attempts to provide an estimation of internet speeds.
-    Requires: speedtest_cli
+    Requires: speedtest-cli/modularize-2
```

Comment:
Maybe add make it a link to the GH branch, because normally "requires" means a PyPI package, while this needs to be installed from source -- you could also show how to install it with pip, e.g. `pip install git+https://github.com/sivel/speedtest-cli.git@modularize-2` (or so)